packages/ckeditor5-table/src/utils/common.ts

```diff
@@ -102,3 +102,14 @@ export function getSelectionAffectedTable( selection: DocumentSelection ): Eleme
 
 	return selection.getFirstPosition()!.findAncestor( 'table' )!;
 }
+
+/**
+ * Based on selected table cells, checks if the table is a content table or not.
+ *
+ * @param selectedCells Selected table cells.
+ */
+export function isTableTypeContent( selectedCells: Array<Element> ): boolean {
+	const table = selectedCells[ 0 ].findAncestor( 'table' )!;
+
+	return table.getAttribute( 'tableType' ) !== 'layout';
+}
```

Comment:
This looks a bit overcomplicated for a single-purpose helper. Please use other common helpers or inline attribute checks where needed.